python/cuml/dask/neighbors/nearest_neighbors.py

```diff
@@ -13,13 +13,16 @@
 # limitations under the License.
 #
 
-from cuml.dask.common import to_dask_cudf, extract_ddf_partitions, \
+from cuml.dask.common import extract_ddf_partitions, \
```

Comment:
While we are making modifications to compound imports like this, we should also separate them each onto their own lines for readability. This also makes it easier to maintain them and lowers the possibility that adding to them will violate flake8 checks.